### PR TITLE
Hotfix BRC20 assertion contents when data is empty

### PR DESCRIPTION
**Pull request by outofxxx**
Regarding the case of an empty response, there are two modifications:

1. When the response data is empty, the default display will be these seven tokens we supported currently. `["ordi", "sats", "rats", "MMSS", "long", "cats", "BTCs"]` , with all balances set to 0.0. The effect is as shown below. 
```json
{
    "assertions": [
        {
            "and": [
                {
                    "src": "$token",
                    "op": "==",
                    "dst": "$ordi"
                },
                {
                    "src": "$holding_amount",
                    "op": ">=",
                    "dst": "0"
                },
                {
                    "src": "$holding_amount",
                    "op": "<",
                    "dst": "1"
                }
            ]
        },
        {
            "and": [
                {
                    "src": "$token",
                    "op": "==",
                    "dst": "$sats"
                },
                {
                    "src": "$holding_amount",
                    "op": ">=",
                    "dst": "0"
                },
                {
                    "src": "$holding_amount",
                    "op": "<",
                    "dst": "40000000"
                }
            ]
        },
        {
            "and": [
                {
                    "src": "$token",
                    "op": "==",
                    "dst": "$rats"
                },
                {
                    "src": "$holding_amount",
                    "op": ">=",
                    "dst": "0"
                },
                {
                    "src": "$holding_amount",
                    "op": "<",
                    "dst": "40000"
                }
            ]
        },
        {
            "and": [
                {
                    "src": "$token",
                    "op": "==",
                    "dst": "$MMSS"
                },
                {
                    "src": "$holding_amount",
                    "op": ">=",
                    "dst": "0"
                },
                {
                    "src": "$holding_amount",
                    "op": "<",
                    "dst": "20"
                }
            ]
        },
        {
            "and": [
                {
                    "src": "$token",
                    "op": "==",
                    "dst": "$long"
                },
                {
                    "src": "$holding_amount",
                    "op": ">=",
                    "dst": "0"
                },
                {
                    "src": "$holding_amount",
                    "op": "<",
                    "dst": "20"
                }
            ]
        },
        {
            "and": [
                {
                    "src": "$token",
                    "op": "==",
                    "dst": "$cats"
                },
                {
                    "src": "$holding_amount",
                    "op": ">=",
                    "dst": "0"
                },
                {
                    "src": "$holding_amount",
                    "op": "<",
                    "dst": "10000"
                }
            ]
        },
        {
            "and": [
                {
                    "src": "$token",
                    "op": "==",
                    "dst": "$BTCs"
                },
                {
                    "src": "$holding_amount",
                    "op": ">=",
                    "dst": "0"
                },
                {
                    "src": "$holding_amount",
                    "op": "<",
                    "dst": "5"
                }
            ]
        }
    ]
}
```
2. The "value" will always be `true`, because regardless of the balance amount, it will always fall within a certain range.

### CODE REVIEW

File: tee-worker/litentry/core/credentials/src/brc20/amount_holder.rs [hunk 1]
```diff
@@ -27,7 +27,7 @@ const VC_BRC20_AMOUNT_HOLDER_DESCRIPTIONS: &str =
 const VC_BRC20_AMOUNT_HOLDER_TYPE: &str = "Token holding amount list";
 
 // Keep all name in lowercase here by purpose
-const BRC20_TOKENS: [&str; 7] = ["ordi", "sats", "rats", "mmss", "long", "cats", "btcs"];
+const BRC20_TOKENS: [&str; 7] = ["ordi", "sats", "rats", "MMSS", "long", "cats", "BTCs"];
```
Comment (jonalvarezz) [inline tee-worker/litentry/core/credentials/src/brc20/amount_holder.rs]: should we remove the comment on line 29 or are we missing something?

File: tee-worker/litentry/core/credentials/src/brc20/amount_holder.rs [hunk 2]
```diff
@@ -72,30 +79,60 @@ pub trait BRC20AmountHolderCredential {
...
 
+// There may be empty cases
+// case 1 : response_items is empty.
+// case 2 : response_items is not empty, but have no interaction with BRC20_TOKENS
+fn collect_brc20_token_balance(response_items: &[ResponseItem]) -> Vec<Brc20TokenBalance> {
+	let mut pairs = vec![];
+
+	response_items
+		.iter()
+		.filter(|&item| BRC20_TOKENS.contains(&item.tick.as_str()))
```
Comment (jonalvarezz) [inline tee-worker/litentry/core/credentials/src/brc20/amount_holder.rs]: should we compare using lowercases to forget about it? e.g., data provider changes the case for `cats` -> `CATs`

**Review by jonalvarezz — APPROVED** [review]
I left a couple of comments but looking good to me overall. 

btw, the `breaking-change` may not be needed, at least not for clients.

File: tee-worker/litentry/core/credentials/src/brc20/amount_holder.rs [hunk 3]
```diff
@@ -72,30 +79,60 @@ pub trait BRC20AmountHolderCredential {
 
 impl BRC20AmountHolderCredential for Credential {
 	fn update_brc20_amount_holder_credential(&mut self, response_items: &[ResponseItem]) {
-		for item in response_items {
-			if BRC20_TOKENS.contains(&item.tick.to_lowercase().as_str()) {
```
Comment (BillyWooo) [inline tee-worker/litentry/core/credentials/src/brc20/amount_holder.rs:76]: should we keep `to_lowercase()` for simplicity? then the above BRC20_TOEKN doesn't need to change.

**Comment by outofxxx** [comment]
Ok, keep `BRC20_TOKENS` letters in lowercase.

**Pull request merged**